docs/websites/cms/install-and-configure-wordpress-using-wp-cli.md

```diff
@@ -0,0 +1,266 @@
+
+## Introduction
+
+Everyone is probably familiar with WordPress and its famous 5 minute install routine. Its simple and works without fuss. But when you have multiple sites to manage, going over the same routine can take up lot of time which you could have used elsewhere.
+
+This is where WP-CLI comes in. It is a commandline tool which comes with lots of powerful features with which you can install, manage, update and do lots more with WordPress. This tutorial will cover how to install WP-CLI and how to perform some common tasks using it.
+
+## Prerequisites
+Before me move ahead, make sure you have a VPS configured with Ubuntu 14.04 and have installed Apache, PHP and MySQL(LAMP stack) on it. You would also need to setup a non-root user first which would be used to work with WP-CLI. If you haven't done that yet, then you can do so by taking help from the following guides.
+
+* [Getting Started with Linode](/docs/getting-started)
+* [How to Install a LAMP Stack on Ubuntu 14.04](/docs/websites/lamp/how-to-install-a-lamp-stack-on-ubuntu-14-04)
+* [Securing your Server](/docs/security/securing-your-server)
+
+## Installing WP-CLI
+
+WP-CLI is available as a PHP Archive file(phar). You can download it using either wget or curl commands.
+
+	curl -O https://raw.githubusercontent.com/wp-cli/builds/gh-pages/phar/wp-cli.phar
+
+Or
+
+	wget https://raw.githubusercontent.com/wp-cli/builds/gh-pages/phar/wp-cli.phar
+
+The following command will check whether the downloaded copy is working correctly or not.
+
+	php wp-cli.phar --info
+
+You would get an output mentioning the version of WP-CLI installed and some basic information. That means, we can move ahead.
+
+Now wp-cli.phar is still not executable and that's why we had to use php command to run it. You need to make it executable and move it to `/usr/local/bin` so that it can be run directly.
+
+	chmod +x wp-cli.phar
+	sudo mv wp-cli.phar /usr/local/bin/wp
+
+Check if it is working properly.
+
+	wp --info
+
+If you see the script version then it means your WP-CLI script has been installed successfully.
+
+### Activating Bash-Completion
+
+Now it is difficult all commands at once and you might want a quick reference on the command you need. While you can always type `wp` and press enter to see the complete list. But wouldn't it be better if you can point to specific command. Like if you want to know what all sub commands are under the `core` parameter. Bash completion script of WP-CLI can help here. To install the script, make sure you are in the home directory of the user you are using.
+
+	cd /home/user
+	wget https://github.com/wp-cli/wp-cli/raw/master/utils/wp-completion.bash
+
+Now edit the .bash_profile so that it is loaded by the shell everytime you login.
+
+	nano ~/.bash_profile
```

Comment:
At guides of this level we assume the reader has a preferred text editor and knows how to use it. Instead of showing the command to open the file, just show what needs to be edited with a {: .file-excerpt} section.